app/Http/Controllers/StaticContentController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class StaticContentController extends Controller
+{
+    public function homePage()
```

Comment:
Remember to follow up on these automatically generated PHPcs comments. In the exam, there might be a point deduction when we find them in your code.

Reply:
"Missing doc comment" are easy to fix. Just type '/**' + enter in the empty line above the method and your IDE will generate a doc comment block for you